<ui version="4.0" >
 <class>operationScreen</class>
 <widget class="QMainWindow" name="operationScreen" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Okno Zabiegu</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>240</width>
     <height>320</height>
    </rect>
   </property>
   <widget class="ScriptWidget" name="timeView" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="frameShape" >
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="verticalScrollBarPolicy" >
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy" >
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="interactive" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line1" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>125</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line2" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>205</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit" >
    <property name="geometry" >
     <rect>
      <x>120</x>
      <y>100</y>
      <width>106</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="wrapping" >
     <bool>false</bool>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly" >
     <bool>true</bool>
    </property>
    <property name="buttonSymbols" >
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="accelerated" >
     <bool>false</bool>
    </property>
    <property name="correctionMode" >
     <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
    </property>
    <property name="keyboardTracking" >
     <bool>false</bool>
    </property>
    <property name="maximumDateTime" >
     <datetime>
      <hour>0</hour>
      <minute>50</minute>
      <second>59</second>
      <year>2000</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="minimumDateTime" >
     <datetime>
      <hour>0</hour>
      <minute>1</minute>
      <second>0</second>
      <year>2000</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="maximumTime" >
     <time>
      <hour>0</hour>
      <minute>50</minute>
      <second>59</second>
     </time>
    </property>
    <property name="currentSection" >
     <enum>QDateTimeEdit::MinuteSection</enum>
    </property>
    <property name="displayFormat" >
     <string>mm:ss</string>
    </property>
   </widget>
   <widget class="ScriptWidget" name="powerView" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>140</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy" >
     <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
      <horstretch>1</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeIncrement" >
     <size>
      <width>1</width>
      <height>1</height>
     </size>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="frameShape" >
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="verticalScrollBarPolicy" >
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy" >
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="powerEdit" >
    <property name="geometry" >
     <rect>
      <x>120</x>
      <y>160</y>
      <width>106</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols" >
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="suffix" >
     <string>%</string>
    </property>
    <property name="minimum" >
     <number>50</number>
    </property>
    <property name="maximum" >
     <number>100</number>
    </property>
    <property name="singleStep" >
     <number>5</number>
    </property>
   </widget>
   <widget class="ScriptWidget" name="levelView" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>220</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="sizePolicy" >
     <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
      <horstretch>1</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeIncrement" >
     <size>
      <width>1</width>
      <height>1</height>
     </size>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="frameShape" >
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="verticalScrollBarPolicy" >
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy" >
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="levelEdit" >
    <property name="geometry" >
     <rect>
      <x>120</x>
      <y>240</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols" >
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="suffix" >
     <string>%</string>
    </property>
    <property name="maximum" >
     <number>100</number>
    </property>
    <property name="singleStep" >
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="selectLabel" >
    <property name="geometry" >
     <rect>
      <x>150</x>
      <y>290</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="frameShape" >
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text" >
     <string>Start</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="backLabel" >
    <property name="geometry" >
     <rect>
      <x>150</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="frameShape" >
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth" >
     <number>1</number>
    </property>
    <property name="midLineWidth" >
     <number>0</number>
    </property>
    <property name="text" >
     <string>Wyjście</string>
    </property>
    <property name="scaledContents" >
     <bool>false</bool>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>120</x>
      <y>80</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="timeLabel" >
       <property name="text" >
        <string>Czas zabiegu:</string>
       </property>
       <property name="buddy" >
        <cstring>timeEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>120</x>
      <y>140</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QLabel" name="powerLabel" >
       <property name="text" >
        <string>Intensywność:</string>
       </property>
       <property name="buddy" >
        <cstring>powerEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>120</x>
      <y>220</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <widget class="QLabel" name="levelLabel" >
       <property name="text" >
        <string>Poziom azotu:</string>
       </property>
       <property name="buddy" >
        <cstring>levelEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
    <zorder>levelLabel</zorder>
    <zorder>horizontalSpacer</zorder>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4" >
     <item>
      <spacer name="horizontalSpacer_4" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>23</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="menuNameLabel" >
       <property name="font" >
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>Zabieg</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>28</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScriptWidget</class>
   <extends>QGraphicsView</extends>
   <header>scriptwidget.h</header>
   <slots>
    <slot>setPName(QString)</slot>
    <slot>setPValue(int)</slot>
    <slot>setPValue(QString)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>selectLabel</tabstop>
  <tabstop>timeEdit</tabstop>
  <tabstop>powerEdit</tabstop>
  <tabstop>levelEdit</tabstop>
  <tabstop>backLabel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>powerEdit</sender>
   <signal>valueChanged(int)</signal>
   <receiver>operationScreen</receiver>
   <slot>powerValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>120</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel" >
     <x>19</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeEdit</sender>
   <signal>timeChanged(QTime)</signal>
   <receiver>operationScreen</receiver>
   <slot>timeValue(QTime)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>163</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel" >
     <x>178</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>levelValue(int)</slot>
  <slot>powerValue(int)</slot>
  <slot>timeValue(QTime)</slot>
 </slots>
</ui>
